### DISCUSSION TITLE
[Feature Request] Slide out Tab bar like Arc

### DISCUSSION THREAD

**Discussion by shaqaruden** [zen-browser/desktop#1901] [Ideas]
I would like to see the option to have the tab sidebar collapse and show like it does in Arc. Meaning when "collapsing" the sidebar instead of collapsing down to just the tab icons to create a navigation rail like panel, I would like to see the sidebar just slide off to the edge and show when I put my mouse on that edge of the window

**Comment by Axenide** [comment]
Have you tried compact mode?

**Reply by shaqaruden** [reply]
that is what I am looking for. I thought compact mode was a navigation rail style sidebar that just took up as little space as possible.
Thanks!
